hack/deploy-hub-local/sno-metal3.yml

```diff
@@ -0,0 +1,23 @@
+version: stable
+tag: 4.10
```

Comment:
you can set tag to 4.10.6 if you want a specific version (to match your spokes.yaml file)

Reply:
it doesn't matter because we're gonna overwrite with the -P param in the call to the plan